### DISCUSSION TITLE
Issue with contributing project.

### DISCUSSION THREAD

**Discussion by pardhu2003** [community/community#134601] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

How can I participate in contributing to this project?

**Comment by kiran4204** [comment, marked as answer]
Contributions are highly welcomed. You can contribute by reviewing and submitting pull requests, reporting issues, suggesting new features, and enhancing the documentation. For detailed guidelines, please refer to the CONTRIBUTING.md file.
